helper/restore_helper.go

```diff
@@ -118,12 +127,47 @@ func (r *RestoreReader) copyAllData() (int64, error) {
 	return bytesRead, err
 }
 
+func (r *RestoreReaderImpl) getReaderType() ReaderType {
+	return r.readerType
+}
+
+func (r *RestoreReaderImpl) getFileHandle() *os.File {
+	return r.fileHandle
+}
+
+func closeAndDeletePipe(h Helper, tableOid int, batchNum int) {
```

Comment:
why not
```suggestion
func (h *Helper) closeAndDeletePipe(tableOid int, batchNum int) {
```

Reply:
This function was taken from ADBDEV-6012 and intentionally left unchanged.

Reply:
[There](https://github.com/arenadata/gpbackup/pull/103/files#diff-f256ea6ee01b09759ff55076ff5ce07cfb4772ec77cd9554e665aa7be5f32332R121) this function didn't have a third argument, so why not make it more correct here?